apps/pie-storybook/.storybook/preview.js

```diff
@@ -55,6 +55,9 @@ export default {
                     value: '#262626',
                 },
             ]
+        },
+        controls: {
+            expanded: true,
```

Comment:
What does this do, out of interest?

Reply:
That shows the description and default value columns in the Controls tab

Reply:
What Xander said – I've added the detail for the button (as that was the component I was updating). Would be good to do the same for the other components too as we update them (as doesn't take v long)